README.md

```diff
@@ -78,3 +78,16 @@ You can then run `docker-compose up` (without `-d`) so that you can check in the
 - Then use this command `migrate -path ./migrations -database "postgres://postgres:password@localhost:5432/postgres?sslmode=disable" force {last-successful-version}`.
 - After that delete the up and down of your failed migration and recreate another migration by `migrate create -ext sql -dir migrations/ you_migration_subject`.
 - And you should be good to go.
+
+### Running fuse project on a windows machine
+
+For this task you will need to use WSL(Windows subsystem for linux) and modify settings in docker desktop application. 
+
+ - Modify your current docker desktop settings to ones showed in the pictures.
+
+<img alt="MergeStat docker desktop general settings" src="docs/docker-general-settings.png" />
+<img alt="MergeStat docker desktop resources settings" src="docs/docker-resources-settings.png" />
+
+ - Follow the instructions provided in this link to make correct use of wsl https://learn.microsoft.com/en-us/windows/wsl/setup/environment
+ - After you are done installing  wsl ,  go ahead and try to install all dependencies needed to run our project inside the wsl 
```

Comment:
```suggestion
 - After you are done installing WSL, go ahead and try to install all dependencies needed to run our project inside the WSL
```

Should we say just try to run `docker-compose up` here using the WSL? Or is there something different that should happen?